backend/substrapp/tasks/tasks.py

```diff
@@ -32,6 +32,8 @@
 COMPOSITE_TRAINTUPLE_TYPE = 'compositeTraintuple'
 TESTTUPLE_TYPE = 'testtuple'
 
+TRAIN_LIKE_TUPLE_TYPES = (TRAINTUPLE_TYPE, AGGREGATETUPLE_TYPE, COMPOSITE_TRAINTUPLE_TYPE)
```

Comment:
Is still relevant to have `TYPE`  in those variables
`TRAIN_LIKE_TUPLE_TYPES` could be simplified as something linked with the local volume to make it explicit

Reply:
Yes I think it is relevant to add the  `_TYPE` suffix to these global variables: it defines the type of tuples returned by the ledger.

If that's ok with you, I will remove the `TRAIN_LIKE_TUPLE_TYPES` as it seems confusing.